pkg/controller/cloneset/cloneset_controller.go

```diff
@@ -99,7 +101,9 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 // add adds a new Controller to mgr with r as the reconcile.Reconciler
 func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// Create a new controller
-	c, err := controller.New("cloneset-controller", mgr, controller.Options{Reconciler: r, MaxConcurrentReconciles: concurrentReconciles})
+	c, err := controller.New("cloneset-controller", mgr, controller.Options{
+		Reconciler: r, MaxConcurrentReconciles: concurrentReconciles,
+		RateLimiter: workqueue.NewItemExponentialFailureRateLimiter(util.BaseDelay, util.MaxDelay)})
```

Comment:
The default rate limiter is https://github.com/openkruise/kruise/blob/master/vendor/k8s.io/client-go/util/workqueue/default_rate_limiters.go#L39 , including `ItemExponentialFailureRateLimiter` and `BucketRateLimiter`.
We should not just use `ItemExponentialFailureRateLimiter` here.

Reply:
done